cypress/integration/fileUpload.spec.ts

```diff
@@ -45,40 +45,37 @@ describe('File upload', () => {
       cy.get('div.btn-group > input[type=file]')
         .attachFile({ filePath: 'acme.png', mimeType: 'image/png' })
       cy.get('.CodeMirror-activeline > .CodeMirror-line > span')
-        .should('have.text', `![](${ imageUrl })`)
+        .should('have.text', `![](${imageUrl})`)
```

Comment:
This seems to be malformatted, because according to our new editor config templates should have white spaces around the content.

Reply:
Hm, I thought I ran the "reformat code" method.

Fixed.